lib/plugins/aws/package/compile/events/httpApi.js

```diff
@@ -88,6 +89,7 @@ class HttpApiEvents {
                     },
                     name: { type: 'string' },
                     scopes: { type: 'array', items: { type: 'string' } },
+                    type: { type: 'string', regexp: new RegExp('^(REQUEST|JWT)$', 'i').toString() },
```

Comment:
I would stick to single case support, and define it here with _enum_.

It think there's no clear benefit in supporting case insensitivity here, while it's  definitely confusing and implementation wise error-prone.

Reply:
I was thinking about it for a moment and I've went with this approach to be consistent with authorizers for `http` event, where we support case insensitvity and even examples in documentation use both notations. Do you think it would be better to introduce that change which will make `httpApi` implementation a bit more consistent while sacrificing consistency between `http` and `httpApi` ?

Reply:
I think that case insensitivity in case of `http` was not great decision. It also predated config based validation.

I would not look back here, same as when deciding on other parts of the config we didn't look back (e.g. in HTTP API it's `provider.httpApi.name` while with Rest API it's `provider.apiName`)

Also if user will migrate from `http` and start to use upper case with `httpApi` it'll get very meaningful error message, so I don't think it should be a problem.

Reply:
:+1: changed